python/py-portalocker/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        WoLpH portalocker 2.2.1 v
+revision            0
+name                py-${github.project}
+license             PSF-2.1.1
+maintainers         nomaintainer
+supported_archs     noarch
+
+description         Cross-platform locking library
+
+long_description    Portalocker is a library to provide an easy API to file locking.
+
+checksums           rmd160  d51dab31282758302fdebb18a4a1f6cdbf28e8bf \
+                    sha256  ee2d983b0c53ee9bf7d01b4a4dde93cbbc60ae2e0edd2420aa196e32a3dd0919 \
+                    size    29804
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    depends_lib-append \
+                    port:py${python.version}-setuptools
+
+    depends_test-append \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-pytest-cov \
+                    port:py${python.version}-pytest-flake8 \
+                    port:py${python.version}-pytest-mypy \
+                    port:py${python.version}-redis \
+                    port:py${python.version}-sphinx
+        
+
+    test.run        yes
+    test.cmd        py.test-${python.branch}
+    test.target     portalocker_tests
+
```

Comment:
are you sure you don't need to set `PYTHONPATH` here (and in other PRs you submitted). In other words, do these tests run like this if you haven't installed the port yet?

Reply:
Thanks fixed.